src/LeanCode.ContractsGenerator.Tests/Notifications/NotificationTagGenerators.cs

```diff
@@ -120,17 +101,15 @@ public void Array()
     }
 
     [Fact]
-    public void Known_type_name_clashes()
+    public void If_a_known_type_clashes_name_with_internal_type_The_clash_is_resolved_correctly()
```

Comment:
```suggestion
    public void If_a_known_type_clashes_name_with_internal_type_the_clash_is_resolved_correctly()
```

Reply:
The capital `T` there is on purpose - it distinguishes first part of the sentence (before ","), from the second one (after ","). Or, it separates "given" part, from "then" part.